examples/README.rst

```diff
@@ -60,13 +60,13 @@ Available Demos for Sparkling Shell
 Building and Running Examples
 -----------------------------
 
-Please see `Running Sparkling Water Examples <../doc/devel/running_examples.rst>`__ for more information how to build
+Please see `Running Sparkling Water Examples <../doc/src/site/sphinx/devel/running_examples.rst>`__ for more information how to build
```

Comment:
The path should not be changed, any reason for that?

Reply:
The current links works and this would probably break it as on the real page, this path does not exist, these are just source files

Reply:
The links here do not work:
https://github.com/h2oai/sparkling-water/tree/master/examples#building-and-running-examples
https://github.com/h2oai/sparkling-water/tree/master/examples#configuring-sparkling-water-variables
If you view the raw file in this branch, the changes make the links work properly.

Reply:
Oh, are those links referenced in the online help somewhere? I only noticed them in github.

Reply:
Ah, my bad. These are links to github repo, not on the regular page. In that case it's a good change, sorry for confusion

Reply:
Sorry, my confusion, I thought it's on the doc page, this related to github. Please go ahead with this link change.

Thank you!